digiwf-apps/packages/apps/digiwf-tasklist/src/components/UI/AppKeyBindingsDialog.vue

```diff
@@ -0,0 +1,261 @@
+            </v-col>
+            <v-col
+              cols="12"
+              sm="6"
+            >
+              Escapetaste
+            </v-col>
+          </v-row>
+          <v-row>
+            <v-col
+              cols="12"
+              sm="6"
+            >
+              Checkbox aktivieren / deaktivieren
+            </v-col>
+            <v-col
+              cols="12"
+              sm="6"
+            >
+              Leertaste
+            </v-col>
+          </v-row>
+          <v-row>
+            <v-col
+              cols="12"
+              sm="6"
+            >
+              Switch aktivieren / deaktivieren
+            </v-col>
+            <v-col
+              cols="12"
+              sm="6"
+            >
+              Leertaste
+            </v-col>
+          </v-row>
+          <v-row>
+            <v-col
+              cols="12"
+              sm="6"
+            >
+              Zahleneingabe
+            </v-col>
+            <v-col
+              cols="12"
+              sm="6"
+            >
+              Tastatureingabe oder Pfeiltasten
+            </v-col>
+          </v-row>
+          <v-row>
+            <v-col
+              cols="12"
+              sm="6"
+            >
+              Datumseingabe
+            </v-col>
+            <v-col
+              cols="12"
+              sm="6"
+            >
+              Tastatureingabe oder Pfeiltasten
+            </v-col>
+          </v-row>
+          <v-row>
+            <v-col
+              cols="12"
+              sm="6"
+            >
+              Uhrzeiteingabe
+            </v-col>
+            <v-col
+              cols="12"
+              sm="6"
+            >
+              Tastatureingabe oder Pfeiltasten
+            </v-col>
+          </v-row>
+          <v-row>
+            <v-col
+              cols="12"
+              sm="6"
+            >
+              Dateieingabe Feld öffnen
+            </v-col>
+            <v-col
+              cols="12"
+              sm="6"
+            >
+              Enter
+            </v-col>
+          </v-row>
+          <v-row>
+            <v-col
+              cols="12"
+              sm="6"
+            >
+              Dateieingabe Feld verlassen ohne Eingabe
+            </v-col>
+            <v-col
+              cols="12"
+              sm="6"
+            >
+              Escapetaste
+            </v-col>
+          </v-row>
+        </v-container>
+      </v-card-text>
+      <v-card-actions>
+        <v-spacer />
+        <v-btn
+          color="blue darken-1"
+          text
+          aria-label="Tastaturbedienungsanleitung schließen"
+          @click="$emit('close')"
+        >
+          Schließen
+        </v-btn>
+      </v-card-actions>
+    </v-card>
+  </v-dialog>
+
+</template>
+
+<script lang="ts">
+import {Component, Emit, Prop, Vue} from "vue-property-decorator";
+
+@Component
+export default class AppKeyBindingsDialog extends Vue {
```

Comment:
Kannst du da als Composition umschreiben. Das brauchen wir für den Umstieg auf vue3. 

Hier ein Beispiel dazu: 
https://github.com/it-at-m/digiwf-core/blob/dev/digiwf-apps/packages/apps/digiwf-tasklist/src/components/task/AssignTaskDialog.vue

Reply:
Als Composition umgeschrieben.